### ISSUE TITLE
Link BAR models to Hugging Face paper page

### ISSUE THREAD

**Issue by NielsRogge** [amazon-far/BAR#2]
Hi @yucornetto 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your excellent work "Autoregressive Image Generation with Masked Bit Modeling" through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2602.09024.
The paper page lets people discuss your paper and find related artifacts (like your models), you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I noticed from your GitHub README that you've released several pre-trained BAR model checkpoints (BAR-B/4, BAR-B/2, BAR-B, BAR-L, BAR-L-res512) and that they are already hosted on the Hugging Face Hub at `FAR-Amazon/BAR-collections`. That's fantastic!

To further improve their visibility and discoverability, I'd love to help you link these models to your paper page. This way, users can easily find and access your models directly from the paper's dedicated page on Hugging Face. We can also ensure appropriate tags are added to the model cards so that people find them more easily when filtering `https://huggingface.co/models`.

If these models are currently within a single collection, we could also discuss creating individual model repositories for each checkpoint. This often provides clearer access, enables specific documentation for each variant, and allows for better tracking of download statistics.

If you're interested, you can find a guide on how to link models to a paper [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper).

Let me know if you're interested or need any guidance on this!

Kind regards,

Niels
